App.tsx

```diff
@@ -1,31 +1,22 @@
 import React from 'react';
-import { applyMiddleware, createStore } from 'redux'
 import { Provider } from 'react-redux'
 import thunk from 'redux-thunk'
-
+import { applyMiddleware, createStore } from 'redux'
 import mainReducer from './redux/reducers/mainReducer'
 import { NavigationContainer } from '@react-navigation/native';
+import MainStackNavigator from './navigation/MainStackNavigator';
+import {FC} from 'react';
 
-import {
-  StyleSheet,
-} from 'react-native';
-import { MainStackNavigator } from './navigation/MainStackNavigator';
-
-const globalStore = createStore(mainReducer, applyMiddleware(thunk))
-
-const App = () => {
+const store = createStore(mainReducer, applyMiddleware(thunk))
 
+const App: FC = (): JSX.Element => {
```

Comment:
[Minor] Don't use FC and JSX.Element together, use FC if the component has props, else use JSX